osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -151,23 +151,27 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
         }
 
         private ScheduledDelegate pushDebounce;
-        private VisualSettings visualSettings;
+        protected VisualSettings VisualSettings;
 
         private bool readyForPush => player.LoadState == LoadState.Ready && IsHovered && GetContainingInputManager()?.DraggedDrawable == null;
 
         protected override bool OnHover(HoverEvent e)
         {
             // restore our screen defaults
             InitializeBackgroundElements();
+            if (this.IsCurrentScreen() && (Background?.IsLoaded ?? false))
+                Background.EnableUserDim.Value = false;
             return base.OnHover(e);
         }
 
         protected override void OnHoverLost(HoverLostEvent e)
         {
-            if (GetContainingInputManager()?.HoveredDrawables.Contains(visualSettings) == true)
+            if (GetContainingInputManager()?.HoveredDrawables.Contains(VisualSettings) == true)
             {
                 // show user setting preview
```

Comment:
update comment to mention that `UpdateBackgroundElements` is only being called because blur hasn't been moved to `UserDimContainer` (and maybe link the issue mentioning that it needs to be moved).